libraries/lib-parameters/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+#[[
+Facilities for defining and visiting the heterogenously typed parameters of an
+effect, or command, etc.
+]]
+
+set( SOURCES
+   Shuttle.cpp
+   Shuttle.h
+   ShuttlePrefs.cpp
+   ShuttlePrefs.h
+   WrappedType.cpp
+   WrappedType.h
+)
+set( LIBRARIES
+   lib-preferences-interface
+   wxBase
+)
+audacity_library( lib-parameters "${SOURCES}" "${LIBRARIES}"
```

Comment:
Probably `lib-shuttle-parameters`? I would never expect to see Shuttle.h in a library called `parameters`

Reply:
"Shuttle" is some not very descriptive terminology from James.  I would call it "ParameterVisitor" if I wanted to fix terminology too instead of just moving files.

Reply:
`lib-parameters` is a way to generic name in any case

Reply:
Macro commands and effects both use them.  Lib-effect-parameters would be too specific.  What is a good but not long name?  I don’t like the legacy term “shuttle”